README.md

```diff
@@ -7,7 +7,7 @@ Node FHIR client library
 
 # Features
 
-* Node
+* Support for STU3 (3.0.1) and DSTU2 (1.0.2)
```

Comment:
Should we say DSTU2 (1.0.2) and up?

Reply:
Actually, versions 1.02, 1.1.0, 1.4.0, 1.6.0, 1.8.0, and 3.0.1 all contain different resources. I could make a separate regex for each version, but with this many, it may be best just to use the union of all possible resource types rather than potentially have to check a reference against six large regexes.

Reply:
Okay. But as far as the README goes, instead of listing out every single FHIR version, can we say:

```
* Support for DSTU2 (1.0.2) and later
```

Reply:
Or actually I am reconsidering it. I think listing them all out shows that the library is contemporary and staying up to date.